index.js

```diff
@@ -122,22 +122,26 @@ class CMSClient {
       const resourceData = await fetch(this.url);
       const headers = await resourceData.headers;
 
-      this.fetched.fields = JSON.parse(
-        headers.get('x-soda2-fields').split(','),
-      );
+      if (headers.get('x-soda2-fields')) {
```

Comment:
Confirming my understanding that this headers is available because CMS data is accessible through a Socrata-based API? Is that correct, @hawyar.

Reply:
Yes, here is the list https://dev.socrata.com/docs/response-codes.html of available headers.